examples/kitty/Makefile

```
@@ -21,21 +22,41 @@ MICROKIT_CONFIG ?= debug
 BUILD_DIR ?= $(abspath build)
 MICROKIT_BOARD := odroidc4
 ETHERNET_CONFIG_INCLUDE ?= src/ethernet_config
-CPU := cortex-a55
+
+ifeq ($(strip $(MICROKIT_BOARD)), odroidc4)
+	CPU := cortex-a55
+	MICROKIT_BOARD := odroidc4
+	SDDF_ETHERNET := meson
+	SDDF_SERIAL := meson
+	SDDF_I2C := meson
+	SDDF_TIMER := meson
+	IMAGES := timer.elf vmm.elf ethernet.elf micropython.elf nfs.elf eth_copy.elf eth_virt_rx.elf eth_virt_tx.elf uart_driver.elf serial_virt_rx.elf serial_virt_tx.elf i2c_virt.elf i2c_driver.elf
+else ifeq ($(strip $(MICROKIT_BOARD)), qemu_arm_virt)
+	CPU := cortex-a53
+	MICROKIT_BOARD := qemu_arm_virt
+	SDDF_ETHERNET := virtio
+	SDDF_SERIAL := arm
+	SDDF_TIMER := arm
+	IMAGES := timer.elf vmm.elf ethernet.elf micropython.elf nfs.elf eth_copy.elf eth_virt_rx.elf eth_virt_tx.elf uart_driver.elf serial_virt_rx.elf serial_virt_tx.elf
+else
+$(error Unsupported MICROKIT_BOARD given)
+endif
 
 TOOLCHAIN := clang
 CC := clang
 LD := ld.lld
 TARGET := aarch64-none-elf
 MICROKIT_TOOL ?= $(MICROKIT_SDK)/bin/microkit
 DTC := dtc
+QEMU := qemu-system-aarch64
 
 BOARD_DIR := $(MICROKIT_SDK)/board/$(MICROKIT_BOARD)/$(MICROKIT_CONFIG)
+SYSTEM_FILE := board/$(MICROKIT_BOARD)/kitty.system
 LIONSOS ?= ../..
 SDDF := $(LIONSOS)/dep/sddf
 LIBVMM := $(LIONSOS)/dep/libvmm
 
-VMM_IMAGE_DIR := src/vmm/images
+VMM_IMAGE_DIR := board/$(MICROKIT_BOARD)/vmm_images
```

```suggestion
FRAMEBUFFER_VMM_IMAGE_DIR := board/$(MICROKIT_BOARD)/framebuffer_vmm_images
```

Not really related to this PR but since we're changing it anyways we should probably make it obvious what it is